File: examples/broken-script.js
// Example: A broken Node.js script to test oops
// Run: node examples/broken-script.js 2>&1 | oops

const express = require('express'); // Module not installed
const app = express();

app.get('/', (req, res) => {
  res.send('Hello World');
});

app.listen(3000);

File: examples/broken-script.py
#!/usr/bin/env python3
# Example: A broken Python script to test oops
# Run: python examples/broken-script.py 2>&1 | oops

from flask import Flask  # Module not installed

app = Flask(__name__)

@app.route('/')
def hello():
    return 'Hello World'

if __name__ == '__main__':
    app.run()
